README.md

```diff
@@ -120,7 +120,8 @@ Tags:
 	document := markdown.Parser().Parse(text.NewReader([]byte(source)))
```

Comment:
@yuin Could you also please help me understand where `text` variable comes from? (I am new to Golang, so I think I am missing something)

Reply:
Understood it comes from `"github.com/yuin/goldmark/text"`